app/Console/Commands/SendTerrafundReportRemindersCommand.php

```diff
@@ -32,10 +32,13 @@ public function handle(): int
             ->chunkById(100, function ($programmes) {
                 $programmes->each(function ($programme) {
                     if ($programme->users->count()) {
-                        Mail::to($programme->users->pluck('email_address'))->queue(new TerrafundReportReminder($programme->id));
                         $programme->users->each(function ($user) use ($programme) {
+                            Mail::to($user->email_address)
```

Comment:
🤔 I understand why you had to do this, but it's going to increase our email send volume quite a bit to break apart emails that used to go to several people into individual emails. We might need to look into rate limiting our email send jobs, or handling errors from SES and re-trying the send later, if that's not already wired up. Nothing to do about it in this PR, but something for us to keep an eye on in Sentry.

Reply:
sure, will keep in mind